docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
+3. Responses MUST have the same value for the producedAt and thisUpdate times.
```

Comment:
This is a take-it or leave it. RFC5019 makes it clear that 'most' should have this.

The goal here is to ensure there's a reasonably sane window between thisUpdate, nextUpdate, and producedAt, to make sure that CAs aren't pre-producing requests far in the future (since that would be a misleading producedAt), and that the information is current. I'm totally open to finding ways to accomplish this goal.